dwellingly/src/views/addProperty.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react';
+import { Form, Field, Formik } from 'formik';
+import * as Yup from 'yup';
+import { UserContext } from '../App';
+import * as axios from 'axios';
+
+const validationSchema = Yup.object().shape({
+    name: Yup.string()
+        .min(5, "*Name must have at least 2 characters")
+        .max(50, "*Names can't be longer than 100 characters")
+        .required("*Name is required"),
+    address: Yup.string()
+        .required("*Address is required"),
+    city: Yup.string()
+        .required("*City is required"),
+    zipcode: Yup.string()
+        .required("*Zipcode is required"),
+    state: Yup.string()
+        .required("*State is required")
+});
+
+const formHandler = (data, context) => {
+    axios.post('http://localhost:5000/properties', data, { headers: {"Authorization" : `Bearer ${context.user.accessJwt}`} })
```

Comment:
This _looks_ right but I'm getting a 422 with an error saying it's missing the Bearer token. I think "Authorization" should _not_ be in quotes by what I can find online